app/config/parameters.yml.dist

```diff
@@ -45,3 +45,7 @@ parameters:
     fosuser_confirmation: true
 
     from_email: no-reply@wallabag.org
+
+    elastic_search: true
```

Comment:
Elasticsearch is one work, no space, no uppercase S. You should write this "elasticsearch".